src/config/IAppConfig.ts

```diff
@@ -66,6 +66,7 @@ export interface IServerConfig {
     api_cache_limit: number;
     show_hotspot: boolean | undefined;
     show_oncokb: boolean;
+    show_oncokb_cbiocosmic: boolean;
```

Comment:
Update property names after update of backend branch.

Reply:
fixed